h2o-core/src/main/java/water/H2O.java

```diff
@@ -298,6 +301,9 @@ public static void printHelp() {
     /** -login_conf is login configuration service file on local filesystem */
     public String login_conf = null;
 
+    /** -ssl_config path (absolute or relative) to a file containing all SSL related configurations */
+    public String ssl_config = null;
```

Comment:
@mmalohlava I already discussed this with @tomkraljevic some time ago and the current implementation is doing this to some extend (hence those cert related sun.* classes). 

I cannot auto generate the keys/config on a standalone H2O cluster since there's no way we can distribute it in a secure fashion, hence the "manual" ssl config. With Hadoop and Spark you can secure YARN which I use to distribute it.

I can split it into two arguments (one to enable it, one for config file), if you think it's more readable. I thought that `ssl_config` with or without a path to a file is enough.